corehq/apps/ota/views.py

```diff
@@ -181,9 +181,15 @@ def claim(request, domain):
 @require_POST
 @check_domain_migration
 def claim_all(request, domain):
```

Comment:
should this also check access of the logged in user (`request.couch_user`)? Just thinking that you could send any data here and potentially claim cases you shouldn't be able to.

Maybe also check that the user specified in the username param has access to the domain?

Reply:
Updated in 42d9b30ef0e to check both the user in the param and the request user have domain access, I think they both should. Although I think claiming is going to shift to formplayer and this code is going to disappear.